command/volume_create_csi.go

```diff
@@ -0,0 +1,28 @@
+package command
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/hcl/hcl/ast"
+	"github.com/hashicorp/nomad/api"
+)
+
+func (c *VolumeCreateCommand) csiCreate(client *api.Client, ast *ast.File) int {
+	vol, err := csiDecodeVolume(ast)
+	if err != nil {
+		c.Ui.Error(fmt.Sprintf("Error decoding the volume definition: %s", err))
+		return 1
+	}
+
+	vols, _, err := client.CSIVolumes().Create(vol, nil)
+	if err != nil {
+		c.Ui.Error(fmt.Sprintf("Error creating volume: %s", err))
+		return 1
+	}
+	for _, vol := range vols {
+		c.Ui.Output(fmt.Sprintf(
+			"Created external volume %s with ID %s", vol.ExternalID, vol.ID))
+	}
```

Comment:
Is partial failure possible here, and how is it handled? It may make sense to report the external volumes that got created before reporting the failure?